dct_backend/reviews.py

```diff
@@ -0,0 +1,91 @@
+from fastapi import APIRouter, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+from pydantic import BaseModel
+import server
+# from fastapi.testclient import TestClient
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+
+food_db = db.client["food"] # from food collection
+review_collection = food_db["reviews"] # review --> sub collection
+# app = FastAPI()
+router = APIRouter(prefix="/reviews")
+resp = Response
+
+class Reviews(BaseModel):
+    """
+        foodID - the name or id associated with a food item
+
+        rating - 1 to 5 stars
+
+        review - written review
+
+        userame - unique ID for each user of quack app
+    """
+    food_id: str
```

Comment:
change all instances of `food_id` to `food_name`. There's no real "food id" that we can scrape. Technically we can hash it but I don't think that's worth. Just make sure that you always deal with the lower case version of the food name and add a test for this